tests/test_changelog.py

```diff
@@ -1136,6 +1139,91 @@ def test_render_changelog(gitcommits, tags, changelog_content):
     assert result == changelog_content
 
 
+def test_render_changelog_from_default_plugin_values(
+    gitcommits, tags, changelog_content
+):
+    parser = ConventionalCommitsCz.commit_parser
+    changelog_pattern = ConventionalCommitsCz.bump_pattern
```

Comment:
```suggestion
    changelog_pattern = ConventionalCommitsCz.changelog_pattern
```

Even though they're identical now, I think it might be better for us to use `changelog_pattern` here